common4j/src/main/com/microsoft/identity/common/java/AuthenticationConstants.java

```diff
@@ -50,6 +66,26 @@
         public static final String CLIENT_REQUEST_ID = "client-request-id";
     }
 
+    /**
+     * Represents the constants value for Active Directory.
+     */
+    @NoArgsConstructor(access = AccessLevel.PRIVATE)
+    public static final class OAuth2 {
+        /**
+         * String of AAD version.
+         */
+        public static final String AAD_VERSION = "ver";
+
+        /**
+         * Constant for  v1 endpoint
+         */
+        public static final String AAD_VERSION_V1 = ONE_POINT_ZERO;
```

Comment:
I kind of feel like `ONE_POINT_ZERO` reduces readability. I kind of read it better if `"1.0"` was hard coded here...may be that's just me